x/erc20/keeper/keeper_test.go

```diff
@@ -354,10 +306,23 @@ func (suite *KeeperTestSuite) DeployContractDirectBalanceManipulation(name strin
 	return crypto.CreateAddress(suite.address, nonce)
 }
 
+// Commit commits and starts a new block with an updated context.
 func (suite *KeeperTestSuite) Commit() {
-	_ = suite.app.Commit()
+	suite.CommitAndBeginBlockAfter(time.Hour * 1)
+}
+
+// Commit commits a block at a given time. Reminder: At the end of each
+// Tendermint Consensus round the following methods are run
+//  1. BeginBlock
+//  2. DeliverTx
+//  3. EndBlock
+//  4. Commit
+func (suite *KeeperTestSuite) CommitAndBeginBlockAfter(t time.Duration) {
```

Comment:
I think we should refactor all the test suites to include this. We should have a clear test util that we can use without having to duplicate all these boilerplate

Reply:
this can be a separate ticket

Reply:
I agree, I explored this a little in this PR, but decided that it should be handled separately